_assets/compose/wnode-test-cluster/wnode-config.json

```diff
@@ -1,6 +1,7 @@
 {
     "NetworkId": 777,
     "DataDir": "/data/ethereumtest/status",
+    "NoBackupDataDir": "/data/ethereumtest/status",
```

Comment:
What does `NoBackupDataDir` stand for? Why is it prefixed with `No`?

Reply:
Stands for `no` as is `dont`. Because we don't want to save in directories that have the automatic backup by default (in android you can disable it, but they provide a `no_backup` directory, in ios I think it's `Library`).